doc/source/tune/key-concepts.rst

```diff
@@ -121,158 +126,237 @@ Each trial is also associated with one instance of a :ref:`Trainable <trainable-
+check out the :ref:`tune.run() API reference<tune-run-ref>`.
 
 Search Algorithms
 -----------------
 
-To optimize the hyperparameters of your training process, you will want to use a :ref:`Search Algorithm <tune-search-alg>` which will help suggest better hyperparameters.
-
-.. code-block:: python
-
-    # Be sure to first run `pip install bayesian-optimization`
-
-    from ray.tune.suggest import ConcurrencyLimiter
-    from ray.tune.suggest.bayesopt import BayesOptSearch
-
-    # Define the search space
-    config = {
-        "a": tune.uniform(0, 1),
-        "b": tune.uniform(0, 20)
-    }
-
-    # Execute 20 trials using BayesOpt and stop after 20 iterations
-    tune.run(
-        trainable,
-        config=config,
-        metric="score",
-        mode="max",
-        # Limit to two concurrent trials (otherwise we end up with random search)
-        search_alg=ConcurrencyLimiter(
-            BayesOptSearch(random_search_steps=4),
-            max_concurrent=2),
-        num_samples=20,
-        stop={"training_iteration": 20},
-        verbose=2)
-
-
-Tune has SearchAlgorithms that integrate with many popular **optimization** libraries, such as :ref:`Nevergrad <nevergrad>` and :ref:`HyperOpt <tune-hyperopt>`. Tune automatically converts the provided search space into the search
-spaces the search algorithms/underlying library expect.
-
-See the documentation: :ref:`tune-search-alg`.
-
-Trial Schedulers
-----------------
-
-In addition, you can make your training process more efficient by using a :ref:`Trial Scheduler <tune-schedulers>`.
-
-Trial Schedulers can stop/pause/tweak the hyperparameters of running trials, making your hyperparameter tuning process much faster.
-
-.. code-block:: python
-
-    from ray.tune.schedulers import HyperBandScheduler
-
-    # Create HyperBand scheduler and maximize score
-    hyperband = HyperBandScheduler(metric="score", mode="max")
-
-    # Execute 20 trials using HyperBand using a search space
-    configs = {"a": tune.uniform(0, 1), "b": tune.uniform(0, 1)}
-
-    tune.run(
-        MyTrainableClass,
-        config=configs,
-        num_samples=20,
-        scheduler=hyperband
-    )
-
-:ref:`Population-based Training <tune-scheduler-pbt>` and :ref:`HyperBand <tune-scheduler-hyperband>` are examples of popular optimization algorithms implemented as Trial Schedulers.
-
-Unlike **Search Algorithms**, :ref:`Trial Scheduler <tune-schedulers>` do not select which hyperparameter configurations to evaluate. However, you can use them together.
+To optimize the hyperparameters of your training process, you use
+a :ref:`Search Algorithm <tune-search-alg>` which suggests hyperparameter configurations.
+If you don't specify a search algorithm, Tune will use random search by default, which can provide you
+with a good starting point for your hyperparameter optimization.
+
+For instance, to use Tune with Bayesian optimization (make sure to first run ``pip install bayesian-optimization``),
```

Comment:
Technically all of the searchers we have are bayesian optimization. Maybe add "simple" or specify the name of library